src/simplewallet/simplewallet.cpp

```diff
@@ -10361,6 +10366,14 @@ bool simple_wallet::user_confirms(const std::string &question)
    return !std::cin.eof() && command_line::is_yes(answer);
 }
 
+bool simple_wallet::user_confirms_auto_config()
+{
+  message_writer(console_color_red, true) << tr("WARNING: Using MMS auto-config mechanisms is not trustless");
+  message_writer() << tr("A malicious auto-config manager could send you info about own wallets instead of other signers' info");
```

Comment:
This statement isn't clear (at least to me). Does this mean a n-of-3 setup could be hijacked by one of the parties to not include all 3 pubkeys? Based on how multi-sig crypto works in Monero, this would be my guess.

Reply:
Right, the statement alone isn't clear. I hope I will get around to amend the MMS manual with some explanations, but in any case only after the release is broadly available: It's probably not a good idea to reveal how you can cheat before that.